static/defaults.config.php

```diff
@@ -273,6 +273,10 @@
 		// Maximum number of queue items for a single contact before subsequent messages are discarded.
 		'max_contact_queue' => 500,
 
+		// max_feed_items (Integer)
+		// Maximum number of feed items that are fetched and processed. For unlimited items set to 0.
+		'max_feed_items' => 10,
```

Comment:
Will this pick up items where it left off the next time the feed is polled of there are more than 10 items?

Reply:
It is always fetching the newest 10 items.

Reply:
What is the number of items of the problematic feed? Can we raise this to 15 or 20 and still be within a reasonable performance threshold?

Reply:
Yeah, no problem with that. I had feeds with more than 200 items, *that* was intense.